src/components/popup/details/vendors/vendor.jsx

```diff
@@ -41,11 +43,21 @@ export default class Vendor extends Component {
 					<LocalLabel localizeKey='legitimateInterestPurposes'>Legitimate interest purposes</LocalLabel>{': '}
 					{legIntPurposes}{'. '}
 				</span>}
+				{specialPurposes && !!specialPurposes.length &&
+				<span>
+					<LocalLabel localizeKey='specialPurposes'>Special purposes</LocalLabel>{': '}
```

Comment:
Can we bold every group name, please?

Reply:
As far as I know the groupNames are bolded due to on-site css'es